README.md

```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
 
     
 
+## Backup and restore of Redis
 
+1. Create a directory to store the backup rdb files and append-only files.
 
+   ```
+   mkdir backup
+   ```
+
+2. Get the name of the master pod to collect the backup snapshot file and appendonly file to restore them later (either ondisk or inmemory database).
+
+   ```
+   kubectl get pods -nodim | grep redis | grep primary
+   ```
+   
+   **Output**:
+
+   ```
+   redis-ha-inmemory-primary-0
+   ```
+   ```
+   redis-ha-ondisk-primary-0
+   ```
+
+3. Copy the rdb files and aof files from the master pod.
+
+   ```
+   kubectl cp odim/<pod-name>:/redis-data/dump.rdb /home/bruce/backups/dump.rdb
+   ```
```

Comment:
where ever there is /home/bruce is present, replace  with the following

<backup dir>/

for e.g. /home/bruce/backups/dump.rdb  

change to

<backup dir>/backups/dump.rdb

Reply:
fixed